src/app/options.cpp

```diff
@@ -312,6 +366,27 @@ QBtCommandLineParameters parseCommandLine(const QStringList &args)
             else if (arg == CONFIGURATION_OPTION) {
                 result.configurationName = CONFIGURATION_OPTION.value(arg);
             }
+            else if (arg == SAVE_PATH_OPTION) {
+                result.savePath = SAVE_PATH_OPTION.value(arg);
+            }
+            else if (arg == STATUS_OPTION) {
+                result.torrentStatus = STATUS_OPTION.value(arg);
+                if (result.torrentStatus != "paused" && result.torrentStatus != "started")
+                    throw CommandLineParameterError(QObject::tr("%1 must specify either started or paused.")
```

Comment:
Good catch. I think `started` and `paused` had better remain consistent regardless of system language otherwise scripts that execute in one language will fail in another.